docs/develop/dotnet/observability.mdx

```diff
@@ -0,0 +1,106 @@
+---
+id: observability
+title: Observability .NET SDK feature guide
+sidebar_label: Observability
+description: Improve observability in your .NET based Temporal Workflows. View which Workflow Executions are tracked by the Temporal Platform and the state of any Workflow Execution.
+toc_max_heading_level: 4
+keywords:
+  - client
+  - code sample
+  - developer guide
+  - guide context
+  - how to
+  - logging
+  - dotnet
+  - dotnet sdk
+  - sdk
+  - search attribute
+  - workflow
+  - observability
+tags:
+  - client
+  - code-sample
+  - developer-guide
+  - guide-context
+  - how-to
+  - logging
+  - dotnet
+  - dotnet-sdk
+  - sdk
+  - search-attribute
+  - workflow
+  - observability
+---
+
+The observability section of the Temporal Developer's guide covers the many ways to view the current state of your [Temporal Application](/temporal#temporal-application)—that is, ways to view which [Workflow Executions](/workflows#workflow-execution) are tracked by the [Temporal Platform](/temporal#temporal-platform) and the state of any specified Workflow Execution, either currently or at points of an execution.
+
+This section covers features related to viewing the state of the application, including:
+
+- [Metrics](#metrics)
+- [Tracing](#tracing)
+- [Logging](#logging)
+- [Visibility](#visibility)
+
+## How to emit metrics {#metrics}
+
+**List of metrics capable of being emitted**
+
+<!-- CONTENT-->
```

Comment:
This appears to be brand new for .NET. I don't think .NET is where we should introduce this new bold heading, it should be done consistently.